snapcraft/cli/parts.py

```diff
@@ -25,8 +28,13 @@ def partscli(ctx):
 
 @partscli.command()
 @click.pass_context
-def update(ctx):
+def update(ctx, **kwargs):
     """Updates the parts listing from the cloud."""
+    container_config = env.get_container_config()
+    if container_config.use_container:
+        project_options = get_project_options(**kwargs)
+        lifecycle.containerbuild('update', project_options, container_config)
```

Comment:
This looks good to me, but I'm wondering why do we have to update the parts in the host?

Reply:
Because `snapcraft define` and `snapcraft search` operate on the host. There's little reason to start a container for those... or do you think we should?